datamodel.prisma

```diff
@@ -54,7 +54,8 @@ enum QuoteTemplate {
 enum ProjectStatus {
 	DRAFT
 	ONGOING
-	FINISHED
+	ARCHIVED
```

Comment:
I think we still have finished projects in the db, we should keep FINISHED and migrate to ARCHIVED. We'll delete this status in the next release.

```suggestion
	FINISHED # deprecated
	ARCHIVED
```